STAR/draft-ietf-acme-star.md

```diff
@@ -424,7 +429,12 @@ Example directory object advertising STAR support with one day star-min-cert-val
 ## Fetching the Certificates
 {: #fetching-certificates}
 
-The certificate is fetched from the certificate endpoint, as per {{I-D.ietf-acme-acme}}, Section 7.4.2.
+The certificate is fetched from the certificate endpoint as per
+{{I-D.ietf-acme-acme}} Section 7.4.2, unless client and server have
+successfully negotiated the "unauthenticated GET" option described in
+{{certificate-get-nego}}.  In such case, the client can simply issue a GET to
+the certificate resource without authenticating herself to the server as
```

Comment:
herself->itself - the client is not a person.

Reply:
And maybe say explicitly "POST-as-GET" in addition to the reference to 7.4.2.

Reply:
ok, done (both).